pylint/test/functional/useless_else_on_loop.py

```diff
@@ -67,7 +67,8 @@ def test_break_in_orelse_deep():
                 if 3 < 2:
                     break
             else:
-                break
+                if 1 < 2:
```

Comment:
@fantix Can we add a separate test for this?

Reply:
Yeah sure thing!